plugin/backend_linux.go

```diff
@@ -641,7 +641,10 @@ func (pm *Manager) Remove(name string, config *types.PluginRmConfig) error {
 
 	removeDir := pluginDir + "-removing"
```

Comment:
Isn't this exactly what we're doing already in this PR?
I can change this to check if the origin dir does not exist when the error is an "IsExist" error.